app/src/main/java/com/pajato/android/gamechat/main/PaneManager.java

```diff
@@ -73,19 +74,21 @@ public void init(final AppCompatActivity context) {
         titleList.add(context.getString(R.string.chat));
         titleList.add(context.getString(R.string.game));
         ViewPager viewPager = (ViewPager) context.findViewById(R.id.viewpager);
+
+        fragmentList.add(new ChatFragment());
```

Comment:
I have a hunch that fragmentList is only relevant in the smartphone layout.  Therefore these two operations should be associated with code that only runs on the non-tablet case.